userguide/network.rst

```diff
@@ -170,272 +169,153 @@ Interfaces
-   | Options             | string         | Additional parameters from                                                                                |
-   |                     |                | `ifconfig(8) <https://www.freebsd.org/cgi/man.cgi?query=ifconfig>`__.                                     |
-   |                     |                | Separate multiple parameters with a space. For example: *mtu 9000* increases the MTU for interfaces       |
-   |                     |                | which support jumbo frames. See :ref:`this note <LAGG_MTU>` about MTU and lagg interfaces.                |
-   |                     |                |                                                                                                           |
-   +---------------------+----------------+-----------------------------------------------------------------------------------------------------------+
-   | IP Address          | numeric        | Enter a static IPv4 or IPv6 address. Example: *10.0.0.3*. Click the drop-down arrow and select a subnet.  |
-   |                     |                | Additional IP address can be added to the interface by clicking                                           |
-   |                     |                | :guilabel:`ADD ADDITIONAL ALIAS`.                                                                         |
-   |                     |                |                                                                                                           |
-   +---------------------+----------------+-----------------------------------------------------------------------------------------------------------+
-
+   +---------------------+----------------+-------------+-----------------------------------------------------------------------------------------------------------+
+   | Setting             | Value          | Type        | Description                                                                                               |
+   |                     |                |             |                                                                                                           |
+   +=====================+================+=============+===========================================================================================================+
+   | Type                | drop-down menu | All         | Choose the type of interface. *Bridge* creates a logical link between multiple networks.                  |
+   |                     |                |             | *Link Aggregation* combines multiple network connections into a single interface. A virtual LAN (*VLAN*)  |
+   |                     |                |             | partitions and isolates a segment of the connection.                                                      |
```

Comment:
```suggestion
   |                     |                |             | partition and isolates a segment of the connection.                                                       |
```

Reply:
Woops. I must've over looked a piece of that sentence. Disregard this suggestion. The sentence makes sense.